extras/benchmark/Running/benchmark.py

```diff
@@ -0,0 +1,803 @@
+# -*- coding: utf-8 -*-
```

Comment:
what is the difference between ``benchmark.py`` and ``brunel_alpha_nest.py``?

Reply:
I have added docstrings to both scripts.